src/routes/accounts.js

```diff
@@ -78,6 +78,7 @@ export default (apiServer, connectors) => {
   })
 
   apiServer.post('/v1/accounts/create', async req => {
+    allowAccessTo(req, secrets, [{ type: 'admin' }]) // for alpha version only admin can creat accout --will be removed in beta
```

Comment:
What about introducing a configuration option for this? That way, we will be able to use the same code for different products. We will only need to swap the configuration.